kratos/input_output/vtk_output.cpp

```diff
@@ -17,84 +17,82 @@
 namespace Kratos
 {
 
-VtkOutput::VtkOutput(ModelPart& rModelPart, Parameters rParameters) : mrModelPart(rModelPart), mOutputSettings(rParameters)
+VtkOutput::VtkOutput(ModelPart& rModelPart, Parameters rParameters)
+    : mrModelPart(rModelPart), mOutputSettings(rParameters)
 {
-    mDefaultPrecision = this->mOutputSettings["output_precision"].GetInt();;
-    this->mDoneTest = false;
-    this->mShouldSwap = false;
-    std::string file_format = this->mOutputSettings["file_format"].GetString();
-    if(file_format == "ascii")
-        mFileFormat =  VtkOutput::FileFormat::VTK_ASCII;
-    else if (file_format == "binary")
+    mDefaultPrecision = mOutputSettings["output_precision"].GetInt();
+    const std::string file_format = mOutputSettings["file_format"].GetString();
+    if (file_format == "ascii") {
```

Comment:
@philbucher  I remember you were against putting those flower brackets when there is only one statement

Reply:
hm sorry, the others convinced me in the meantime

also without it is more error-prone, if someone adds new line

Reply:
:-) ... no worries !